src/phoneField/index.tsx

```diff
@@ -180,39 +194,44 @@ export default class PhoneField extends PureComponent<PhoneFieldProps, PhoneFiel
       textFieldPlaceholder,
       defaultPhoneValue,
       isInline,
+      error,
     } = this.props
 
     const baseClassName = cc([prefix({ phoneField: true })])
+    const wrapperClassName = `${baseClassName}-wrapper`
 
     const classNames = cc([
-      isInline ? `${baseClassName}--inline` : baseClassName,
+      isInline ? `${wrapperClassName}--inline` : wrapperClassName,
       this.props.className,
-      this.state.hasFocus && `${baseClassName}--hasFocus`,
+      this.state.hasFocus && `${wrapperClassName}--hasFocus`,
     ])
 
     return (
-      <div id={id} className={classNames} aria-labelledby={ariaLabelledBy}>
-        <SelectField
-          name={FIELDS.PHONEREGION}
-          options={this.state.countryData}
-          ariaLabel={selectFieldLabel}
-          defaultValue={this.props.defaultRegionValue}
-          onChange={this.handleChange}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
-        <TextField
-          type="tel"
-          name={FIELDS.PHONENUMBER}
-          placeholder={textFieldPlaceholder}
-          defaultValue={defaultPhoneValue}
-          onChange={this.handleChange}
-          title={textFieldTitle}
-          onFocus={this.onFocus}
-          onBlur={this.onBlur}
-          focusBorder={!isInline}
-        />
+      <div className={cc([baseClassName, prefix({ error: !!error })])}>
```

Comment:
Boolean(error) instead of !!

Reply:
For my part I prefer the short syntax for boolean casting, but that's a matter of preferences.

Reply:
I'm ok with both but we already use Boolean below in the same file, being consistent in the same file would be nice. But it's just a nit, I don't care too much too.